tembo-operator/src/apis/coredb_types.rs

```diff
@@ -222,14 +268,21 @@ pub struct Backup {
     #[serde(default = "defaults::default_backup_schedule")]
     pub schedule: Option<String>,
 
-    /// The S3 compatable endpoint URL
+    /// The S3 compatible endpoint URL
     #[serde(default, rename = "endpointURL")]
     pub endpoint_url: Option<String>,
 
     /// The S3 credentials to use for backups (if not using IAM Role)
     #[serde(default = "defaults::default_s3_credentials", rename = "s3Credentials")]
     pub s3_credentials: Option<S3Credentials>,
 
+    /// The Google Cloud Storage credentials to use for backups
+    #[serde(
+        default = "defaults::default_google_credentials",
```

Comment:
Maybe we remove this default, since that may include the default google creds in the case of AWS backups